### DISCUSSION TITLE
Add bob marley!

### DISCUSSION THREAD

**Discussion by Kylescher1** [some9000/StylePile#18] [Ideas]
Bob marley style would be so cool

**Comment by some9000** [comment]
Will do. 289 artists at the moment so I will go for 300.

**Comment by Kylescher1** [comment]
I would also consider increasing the slider range from 0-2 to 0-10

**Comment by some9000** [comment]
You mean artist strength and such? Because 2 is already overkill and causes strong distortions. Usually 1.5 is close to max. You can test it by locking seed and just generating images in sequence while increasing the values. Results go crazy very quickly.

**Reply by Kylescher1** [reply]
Yes, but maybe add another digit maybe. Such like 1.55 0-2 does not have enough values to fine tune.

**Reply by some9000** [reply]
Ok, that makes sense, will do. I did testing with .1 steps and it felt like that's quite enough. But more precision doesn't hurt anyone so might as well.
